install.sh

```diff
@@ -136,7 +136,9 @@ fi
 
 ./install_configure_selinux.sh
 
-./install_configure_sshd.sh
+if ! ./install_configure_sshd.sh; then
+    exit 1
```

Comment:
If we failed to edit the `sshd_config` file, then we probably shouldn't continue.

Reply:
I agree and I was hoping that my `-e` at the very top achieves this?

Reply:
I thought the same. I wasn't seeing the expected behavior in my shell, but I'll test it out again and report back.